ee_tests/src/specs/quickstart_deployments.spec.ts

```diff
@@ -131,33 +131,29 @@ describe('Creating new quickstart in OSIO', () => {
     await spacePipelinePage.spaceHeader.deploymentsOption.clickWhenReady();
 
     await browser.sleep(5000);
+
+    support.info('Verifying deployments page');
+
     let spaceDeploymentsPage = new SpaceDeploymentsPage();
 
-    /* Verify that app deployed to stage presents: Success icon, blue circle graphic, text = "Output from run = 1 pods" */
+    let deploymentContainers = await spaceDeploymentsPage.getDeploymentCardContainers();
```

Comment:
Naming methods according to CSS classes doesn't feel right. I would suggest something like `let applications = await deploymentsPage.getApplications();`